.npmignore

```diff
@@ -15,6 +15,9 @@ bower.json
 ember-cli-build.js
 testem.js
 
+# sandbox examples
+/lib/sandbox
```

Comment:
should this whole dir be ignored?

Reply:
I think so, for the sake of publishing. These sandbox examples should just be available to the dummy app, not anyone who installs, correct?

I don't think this would ignore `/lib`

Reply:
Oh oh npmignore not gitignore 🤦‍♂️

yep!